opensearchpy/_async/client/__init__.py

```diff
@@ -1329,7 +1329,7 @@ async def reindex_rethrottle(self, task_id, params=None, headers=None):
 
     @query_params()
     async def render_search_template(
-        self, body=None, id=None, params=None, headers=None
+            self, body=None, id=None, params=None, headers=None
```

Comment:
Can we have this in the same line as in line 1331?

Reply:
Unintentional change, fixed it thanks!